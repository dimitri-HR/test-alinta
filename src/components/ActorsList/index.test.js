import React from 'react'
import renderer from 'react-test-renderer'

import ActorList from '.'

const mockActors = [
  {
    name: 'Actor1',
    roles: ['Character1', 'Character2'],
  },
  {
    name: 'Actor2',
    roles: ['Character3', 'Character4'],
  },
]

describe('ActorList', () => {
  it('renders without errors', () => {
    const component = renderer.create(<ActorList actors={mockActors}/>)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
