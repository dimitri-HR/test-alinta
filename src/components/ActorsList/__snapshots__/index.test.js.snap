// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActorList renders without errors 1`] = `
<div>
  <p>
    Actor1
  </p>
  <ul>
    <li>
      Character1
    </li>
    <li>
      Character2
    </li>
  </ul>
  <p>
    Actor2
  </p>
  <ul>
    <li>
      Character3
    </li>
    <li>
      Character4
    </li>
  </ul>
</div>
`;
